bottlecap/src/metrics/datadog.rs

```diff
@@ -0,0 +1,148 @@
+//!Types to serialize data into the Datadog API
+
+use std::env;
+
+use ureq;
+use serde::{Serialize, Serializer};
+use serde_json;
+use tracing::error;
+
+/// Interface for the DogStatsD metrics intake API.
+#[derive(Debug)]
+pub struct DdApi {
+    // runtime: tokio::runtime::Runtime,
+    api_key: String,
+}
```

Comment:
FWIW I think you can park a `ureq::Agent` in this struct and avoid needing an Arc etc.